### DISCUSSION TITLE
Problems with spritesheet frame cut

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [axmolengine/axmol#1531] [General]
Hi,

I have a small problem here:
https://github.com/paulocoutinhox/axmol-bomb-test

Im try cut the spritesheet in 4 frames of 78,64.

But i don't know why, it is cutting wrong as you can see on screenshot:

<img width="1392" alt="image" src="https://github.com/axmolengine/axmol/assets/395096/809da89a-ee58-4a88-9369-5a0a5f6ecb06">

Main code:
https://github.com/paulocoutinhox/axmol-bomb-test/blob/main/Source/MainScene.cpp#L5-L32

Thanks.

**Comment by rh101** [comment]
It may be easier to generate a PLIST file to go with it, and load that in instead.

Otherwise, refer to the PLIST implementation to see how it sets up the sprite frames in order to figure out what you're missing in your code.  There is also a simpler JSON sprite sheet example in the cpp-tests project (search for `GenericJsonArraySpriteSheetLoader`).

**Reply by paulocoutinhox** [reply]
I dont understand what is the difference internally, but using plist works.
